### ISSUE TITLE
Wrong line numbers using logging module

### ISSUE THREAD

**Issue by ObiWahn** [IronLanguages/main#1283]
Hi,

I use IronPython with -X:FullFrames and I still get always 1 as line number when using the logging module.
I think it might be related to how sys._getframe() works. Please give me some hint how to fix this. The same code is running without the .NET part fine on linux using CPython.

Regards

**Comment by ObiWahn** [comment]
Another cause of the problem might be imp.load_source that is used in the following way:

```Python
    if customer_configuration_obj.post_path != None:
        sys.path.append(customer_configuration_obj.customer_directory)
        import imp
        import traceback
        customer_module = imp.load_source('customer', customer_configuration_obj.post_path)
        logger.info("Running Customer Post")
        try:
            customer_module.main(operations, machineSetup, cam_data ,additional_data)
            logger.info("Customer Post finished")
        except Exception as e:
            logger.error("Customer Post failed")
            logger.error(traceback.format_exc())
            exc_info = sys.exc_info()
            raise exc_info[0], exc_info[1], exc_info[2]
```

**Comment by talsaiag** [comment]
This is fixed under my pull request: https://github.com/IronLanguages/main/pull/1277
:)

**Comment by slide** [comment]
Fixed in #1277
